src/localization/index.js

```diff
@@ -20,11 +20,16 @@
 
 import en from './en';
 import fi from './fi';
+import pt from './pt';
 import {flattenMessages} from 'utils/locale-utils';
+import {LOCALE_CODES} from './locales';
 
 const en_flat = flattenMessages(en);
 
 export const messages = {
```

Comment:
```suggestion
export const messages = Object.keys(LOCALE_CODES)
.reduce((acc, key) => ({...acc, [key]: key === "en" ? en_flat : {...en_flat, ...flattenMessages(key)}}), {})
```
what do you think @heshan0131 ?